thirdeye-ui/src/app/components/alert-wizard-v2/alert-template/alert-template.utils.ts

```diff
@@ -159,6 +159,25 @@ export function hasRequiredPropertyValuesSet(
     });
 }
 
+export function createNewStartingAlertThreshold(): EditableAlert {
+    return {
+        name: "",
```

Comment:
is it ok to set the name to "data-preview" ? 
some errors returned by ThirdEye contain the name, and I think this name would help to understand what happens.